__tests__/index.js

```diff
@@ -61,19 +61,19 @@ async function execCommand(
 }
 
 function expectAddSuccessfullOutput(stdout, pkg) {
-  const lastLines = stdout.slice(stdout.length - 4);
+  const lastLines = stdout.slice(stdout.length - 7);
   expect(lastLines[0]).toEqual('success Saved lockfile.');
   expect(lastLines[1]).toEqual('success Saved 1 new dependency.');
-  expect(lastLines[2]).toContain(pkg);
-  expect(lastLines[3]).toContain('Done');
+  expect(lastLines[3]).toContain(pkg);
+  expect(lastLines[6]).toContain('Done');
 }
 
 function expectAddSuccessfullOutputWithNoLockFile(stdout, pkg) {
-  const lastLines = stdout.slice(stdout.length - 4);
+  const lastLines = stdout.slice(stdout.length - 7);
   expect(lastLines[0]).not.toEqual('success Saved lockfile.');
   expect(lastLines[1]).toEqual('success Saved 1 new dependency.');
-  expect(lastLines[2]).toContain(pkg);
-  expect(lastLines[3]).toContain('Done');
```

Comment:
Wondering if converting this test to a "snapshot" would be easier than trying to maintain this "index of line of output" approach...

Reply:
So I had to make the tests not run concurrently (otherwise the snapshots would match random stdout depending on the order the commands ran) and still slice the output (remove yarn version and the time it took to complete the command), but I think the change is still worthwhile for developer experience.